BranchMetrics/xamarin-branch-deep-linking-attribution
# [SDK-2316] Updated native SDK and exposed new methods

## Reference
SDK-2316 -- .net maui update underlying native SDKs

## Summary
Updated the Android and iOS SDKs to the latest versions, exposed the new DMA method, and updated the testbed to improve testing.

All of the code/file changes in the `BranchSDK.xcframework` can be ignored since that's just the new imported iOS SDK. Also the `BranchSDK.iOSBinding/ApiDefinitions.cs` contains a lot of diffs from code being re-ordered, but also contains the update for th new method. The rest of the files should include the relevant, manual code changes.

## Motivation

Update the native SDK's to support the new DMA method and support the identity deprecation. 

## Type Of Change


## Testing Instructions

Run the Timber testbed and check that all of the methods and requests look correct.









cc @BranchMetrics/saas-sdk-devs for visibility.
